### ISSUE TITLE
Issues with Skyrim & Fallout 4 with 3.16-6 ED

### ISSUE THREAD

**Issue by fls2018** [redmcg/wine#2]
They were working with your Proton for a short while but now I'm getting "unable to find .ini file" errors.
If I manually add the .ini files to Documents the games won't load and if I clear the prefixes it reverts back to "unable to find .ini files".

Standard 3.16-6 beta appears to be continuing to work fine, if I then revert back to this proton I can't launch the games again.

Maybe something to do with the whole VCredist hack these don't like.

[steam-489830.log](https://github.com/redmcg/wine/files/2705622/steam-489830.log)

**Comment by redmcg** [comment]
I just tried Skyrim and got the same thing.

The TLDR is: the workaround for games other than ED is to use the Steam Beta client.

The long story is that there were changes to GNUTLS in 3.16-6 that used a later version of the header files - so my old build environment wouldn't compile it. As a result I used the new vagrant build environment provided within Proton. I saw that it was using the Steam Beta runtime but I also saw changes to wine that catered for multiple versions of GNUTLS. This led me to believe (incorrectly) that the new build environment would therefore cater for both Steam Beta and the standard run-time.

The good news it looks like ED now runs in both Beta and Standard, but some other titles will only run in Beta.

**Comment by fls2018** [comment]
I've tried the beta client still the same issue, the launchers refuse to start the game:

https://youtu.be/JWfijb4Xpbc

Strange thing it was working earlier, and if I change the Proton it works again.

**Comment by fls2018** [comment]
I managed to get it working again but I literally replaced the Proton file with the stock 3.16-6 one as well as deleting most the extra stuff for auto-downloading .Net & vcredist.

I have tried compiling Proton with your wine for something a little cleaner but experiencing some headaches with that vagrant debian VM at the minute.

**Comment by redmcg** [comment]
Yeah - not working for me under Beta now either.

I tried copying my proton script the other way (in to the Valve provided version) and their version still works. So I still think it's a build problem - but that leaves me with no explanation as to why it now works for you.

**Comment by redmcg** [comment]
OK - now the Valve provided version has stopped working. But I think I've found the problem.

I have a line in my 'proton' script which removes ESYNC from the environment. It's only there for the install of dotnet40. But when I comment out that line, Skyrim seems to run fine (under both the Valve provided version and mine).

My theory is that the environment change seems to persist in to the game running (which wasn't my intention) and it would appear Skyrim needs ESYNC. I might remove that line and encourage people to increase their file descriptor limit (or see if I can restore the old value after the install).

**Comment by redmcg** [comment]
This seems to be fixed in https://github.com/redmcg/Proton/commit/09f5ea82bd5462fbd75d61a9f9e32eac01e842ea

Let me know if this works for you. I seem to be getting consistent results now - but as you can see from my comments; results were inconsistent early (making it hard to pinpoint the issue).

Edit: Forgot to mention it appears Skyrim works fine with ESYNC switched off - so there must be something else that runs that was being affected by turning ESYNC off. I never got to the bottom of it.

**Comment by redmcg** [comment]
I'll close this one - but please do re-open if this new version of the [proton](https://raw.githubusercontent.com/redmcg/Proton/ED_Proton_3.16/proton) script doesn't work for you.
